? addressbook/libebook/libebook-1.2.pc
? addressbook/libebook/libebook.pc
? addressbook/libedata-book/libedata-book-1.2.pc
? addressbook/libedata-book/libedata-book.pc
? calendar/backends/weather
? calendar/libecal/libecal-1.2.pc
? calendar/libecal/libecal.pc
? calendar/libedata-cal/libedata-cal-1.2.pc
? calendar/libedata-cal/libedata-cal.pc
? calendar/libical/src/libicalss/icalssyacc.output
? libedataserver/libedataserver-1.2.pc
? libedataserver/libedataserver.pc
? servers/groupwise/libegroupwise.pc
? src/.server.c.swp
? src/GNOME_Evolution_DataServer_1.2.server
Index: configure.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/configure.in,v
retrieving revision 1.69
diff -u -r1.69 configure.in
--- configure.in	23 Sep 2004 16:45:27 -0000	1.69
+++ configure.in	30 Sep 2004 11:09:42 -0000
@@ -312,6 +312,7 @@
 calendar/backends/groupwise/Makefile
 calendar/backends/http/Makefile
 calendar/backends/contacts/Makefile
+calendar/backends/weather/Makefile
 calendar/tests/Makefile
 calendar/tests/ecal/Makefile
 libdb/Makefile
Index: calendar/backends/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/calendar/backends/Makefile.am,v
retrieving revision 1.3
diff -u -r1.3 Makefile.am
--- calendar/backends/Makefile.am	9 Jan 2004 20:05:27 -0000	1.3
+++ calendar/backends/Makefile.am	30 Sep 2004 11:09:43 -0000
@@ -1 +1 @@
-SUBDIRS = file groupwise http contacts
+SUBDIRS = file groupwise http contacts weather
Index: calendar/libical/src/libicalss/icalssyacc.h
===================================================================
RCS file: /cvs/gnome/libical/src/libicalss/icalssyacc.h,v
retrieving revision 1.12
diff -u -r1.12 icalssyacc.h
--- calendar/libical/src/libicalss/icalssyacc.h	28 May 2004 10:14:24 -0000	1.12
+++ calendar/libical/src/libicalss/icalssyacc.h	30 Sep 2004 11:09:43 -0000
@@ -78,7 +78,7 @@
 typedef union YYSTYPE {
 	char* v_string;
 } YYSTYPE;
-/* Line 1249 of yacc.c.  */
+/* Line 1240 of yacc.c.  */
 #line 82 "icalssyacc.h"
 # define yystype YYSTYPE /* obsolescent; will be withdrawn */
 # define YYSTYPE_IS_DECLARED 1
Index: src/GNOME_Evolution_DataServerLDAP.server.in.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/GNOME_Evolution_DataServerLDAP.server.in.in,v
retrieving revision 1.12
diff -u -r1.12 GNOME_Evolution_DataServerLDAP.server.in.in
--- src/GNOME_Evolution_DataServerLDAP.server.in.in	11 Mar 2004 10:23:24 -0000	1.12
+++ src/GNOME_Evolution_DataServerLDAP.server.in.in	30 Sep 2004 11:09:45 -0000
@@ -37,6 +37,7 @@
 		<item value="webcal"/>
 		<item value="groupwise"/>
 		<item value="contacts"/>
+		<item value="weather"/>
         </oaf_attribute>
 </oaf_server>
 
Index: src/GNOME_Evolution_DataServerNOLDAP.server.in.in
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/GNOME_Evolution_DataServerNOLDAP.server.in.in,v
retrieving revision 1.14
diff -u -r1.14 GNOME_Evolution_DataServerNOLDAP.server.in.in
--- src/GNOME_Evolution_DataServerNOLDAP.server.in.in	11 Mar 2004 10:23:24 -0000	1.14
+++ src/GNOME_Evolution_DataServerNOLDAP.server.in.in	30 Sep 2004 11:09:45 -0000
@@ -36,6 +36,7 @@
 		<item value="webcal"/>
 		<item value="groupwise"/>
 		<item value="contacts"/>
+		<item value="weather"/>
         </oaf_attribute>
 </oaf_server>
 
Index: src/Makefile.am
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/Makefile.am,v
retrieving revision 1.16
diff -u -r1.16 Makefile.am
--- src/Makefile.am	23 Sep 2004 16:45:29 -0000	1.16
+++ src/Makefile.am	30 Sep 2004 11:09:45 -0000
@@ -67,6 +67,7 @@
 	$(top_builddir)/calendar/backends/http/libecalbackendhttp.la		\
 	$(top_builddir)/addressbook/libebook/libebook-1.2.la			\
 	$(top_builddir)/calendar/backends/contacts/libecalbackendcontacts.la	\
+	$(top_builddir)/calendar/backends/weather/libecalbackendweather.la	\
 	$(LDAP_LIBS)								\
 	$(E_DATA_SERVER_LIBS)
 
Index: src/server.c
===================================================================
RCS file: /cvs/gnome/evolution-data-server/src/server.c,v
retrieving revision 1.20
diff -u -r1.20 server.c
--- src/server.c	2 Aug 2004 18:26:18 -0000	1.20
+++ src/server.c	30 Sep 2004 11:09:45 -0000
@@ -56,6 +56,7 @@
 #include <backends/groupwise/e-cal-backend-groupwise.h>
 #include <backends/http/e-cal-backend-http.h>
 #include <backends/contacts/e-cal-backend-contacts.h>
+#include <backends/weather/e-cal-backend-weather.h>
 
 #include "server-interface-check.h"
 #include "server-logging.h"
@@ -239,6 +240,7 @@
 	e_data_cal_factory_register_method (e_data_cal_factory, "webcal", ICAL_VEVENT_COMPONENT, E_TYPE_CAL_BACKEND_HTTP);
 	e_data_cal_factory_register_method (e_data_cal_factory, "webcal", ICAL_VTODO_COMPONENT, E_TYPE_CAL_BACKEND_HTTP);
 	e_data_cal_factory_register_method (e_data_cal_factory, "contacts", ICAL_VEVENT_COMPONENT, E_TYPE_CAL_BACKEND_CONTACTS);
+	e_data_cal_factory_register_method (e_data_cal_factory, "weather", ICAL_VEVENT_COMPONENT, E_TYPE_CAL_BACKEND_WEATHER);
 
 	if (!e_data_cal_factory_register_storage (e_data_cal_factory, E_DATA_CAL_FACTORY_OAF_ID)) {
 		bonobo_object_unref (BONOBO_OBJECT (e_data_cal_factory));
